packages/reaction-product-variant/client/templates/products/productDetail/variants/variantList/variantList.js

```diff
@@ -6,31 +6,38 @@ Template.variantList.helpers({
     let inventoryTotal = 0;
     const variants = ReactionProduct.getTopVariants();
 
-    if (variants.length > 0) {
+    if (variants.length) {
       // calculate inventory total for all variants
-      for (let variant of variants) {
-        let qty = ReactionProduct.getVariantQuantity(variant);
+      variants.forEach(variant => {
```

Comment:
I'm answered [here](https://github.com/reactioncommerce/reaction/pull/950#discussion_r58326180). In my opinion we should use Array iteration methods for iterate Arrays. Think about it like moving project to more ES6-like code style. Is that make sense?